src/iuse.cpp

```diff
@@ -4683,11 +4683,6 @@ int iuse::set_trap(player *p, item *it, bool, point)
         return 0;
     }
 
-    if (dirx == p->posx() && diry == p->posy()) {
```

Comment:
Uh, I'm sure there's a good reason why the is_benign check can't be used here, but off top/head I don't know it.  Help me understand?